iml-agent/src/action_plugins/check_stratagem.rs

```diff
@@ -0,0 +1,5 @@
+use crate::{agent_error::ImlAgentError, rpm};
+
+pub async fn check_stratagem(_: ()) -> Result<bool, ImlAgentError> {
```

Comment:
Will need to be in a differently named file (`rpm_installed.rs`?), but this could be something like;

```rust
pub async fn check_stratagem(package: String) -> Result<bool, ImlAgentError> {
    rpm::query(&package).await
}
```

Reply:
Done